src/main/java/Orders/OrderType.java

```diff
@@ -0,0 +1,7 @@
+package Orders;
+
+public enum OrderType {
+    delivery_order,
```

Comment:
Naming is not by Convention (please find yourself the rules of naming enums and everything else).